Makefile

```diff
@@ -304,6 +304,19 @@ uninstall:
 	rm -rf $(DESTDIR)$(INSTALL_BIN)/kplutus
 	rm -rf $(DESTDIR)$(INSTALL_LIB)/kplutus
 
+# Unit tests
+#-----------
+
+KPROVE := kprove
+
+unit-tests/%.md.run: unit-tests/%.md
+	@echo $(BWhite)Running unit test: $(Color_off)$(Green)$<$(Color_Off)"\n"
+	$(KPROVE) $< --directory $(KPLUTUS_LIB)/$(haskell_dir) --def-module UPLC
```

Comment:
You need to do `kplc prove` instead of `kprove` directly.

And you need to make sure you can do `kplc kompile ...` as well, and make sure that `kplc prove ...` uses `kprovex`.

It's a lot of infrastructure setup, but will pay off.